.github/workflows/ci-python.yml

```diff
@@ -127,14 +127,24 @@ jobs:
         if: startsWith(matrix.os, 'windows-')
         shell: bash
         run: |
+          set -eux
           choco install -y --limit-output winfsp --version=${{ matrix.winfsp-version }}
           mkdir winfsp-test
           curl -L https://github.com/billziss-gh/winfsp/releases/download/v1.8/winfsp-tests-1.8.20304.zip -o winfsp-test/winfsp-tests.zip
           unzip winfsp-test/winfsp-tests.zip -d winfsp-test
-          pwd
-          echo "$(pwd)\winfsp-test" >> "$GITHUB_PATH"
+          echo "${GITHUB_WORKSPACE}/winfsp-test" >> $GITHUB_PATH
         timeout-minutes: 5
 
+      - name: (🏁 Windows) Check that winfsp-tests is available
+        if: startsWith(matrix.os, 'windows-')
+        shell: python
+        run: |
+          from distutils.spawn import find_executable
+          path = find_executable("winfsp-tests-x86")
+          if not path:
+            raise RuntimeError("winfsp-tests-x86 not found")
+          print(path)
```

Comment:
Why not removing the check at https://github.com/Scille/parsec-cloud/blob/1898409da00fb4879afaa453cd417dc770487921/tests/core/mountpoint/test_winfsp_tests.py#L15-L18 ?

Reply:
because it's ok to run the tests without `winfsp-tests`, locally for instance

Reply:
Why not using another markers for tests that require winfsp tests ?

Reply:
Because the winfsp-tests executable is necessary to get the list of the test cases.

If the executable is not available, there are no tests to collect hence no tests to mark.